### ISSUE TITLE
Please post any comments on the pydra notebooks here.

### ISSUE THREAD

**Issue by satra** [nipype/pydra#147]
we would like to collect feedback in one place. if you test things out yourself or through binder, please post any comments here.

**Comment by tsalo** [comment]
I didn't see any examples of tasks made from imported functions (rather than decorators applied at function definition) in the notebooks. For example, to create a task for the import function `numpy.fft.fft`, you need to do:

```python
# To convert an existing function into a task
# NOTE: Does not work with ufuncs
import numpy as np
import pydra
fft = pydra.mark.annotate({'a': np.ndarray, 'return': float})(np.fft.fft)
fft = pydra.mark.task(fft)
```

**Comment by djarecka** [comment]
@tsalo - thank you! will add it!

**Comment by satra** [comment]
@djarecka - has this been added to the tutorial? if so please close this.

**Comment by djarecka** [comment]
I will

**Comment by nicolocin** [comment]
Can't run **DockerTask** in the `intro_shelltask.py` tutorial.  Docker not install properly?   Pretty sure it was working at some point.

![Screen Shot 2020-04-19 at 4 15 46 PM](https://user-images.githubusercontent.com/30167129/79683066-fad22e00-8259-11ea-80ec-40d990413cb3.png)

**Comment by jsmentch** [comment]
I also ran into the same bug as @nicolocin. I'm going through the binder notebooks and everything worked up until DockerTask

**Comment by djarecka** [comment]
Thanks @nicolocin and @jsmentch . I'm updating the tutorials and will make sure that this works!

**Comment by djarecka** [comment]
actually this example will not work with binder, have to make this clear. At least the current docker container we're using with binder doesn't run docker inside.

@nicolocin - you didn't say where are you running the notebook. Could you please confirm that this was with binder as well?

**Comment by nicolocin** [comment]
@djarecka yes got that error on binder!  If it doesn't work with binder should we exclude that from the tutorial then?   Or at least a note saying that for now `DockerTask`  only works locally

**Comment by tclose** [comment]
The notebooks have been rewritten
